src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -1751,6 +1752,10 @@ def forward(
         return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
         if labels is not None:
+            if labels.shape[1] > 448:
```

Comment:
448 is hardcoded, which is something we would probably like to avoid. Do you think you could get this value from the config?

I think [config.max_length](https://huggingface.co/openai/whisper-large-v3/blob/main/config.json#L36) should do the trick